tests/Unit/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/Test_FishboneMoncriefDisk.cpp

```diff
@@ -162,6 +188,13 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.AnalyticSolutions.RelEuler.FMDisk",
   test_variables(DataVector(5));
 }
 
+SPECTRE_TEST_CASE(
```

Comment:
Please write this into a function in an anonymous namespace and call that from an existing test case. In general, we need to reduce the number of `SPECTRE_TEST_CASE`s by doing this sort of refactoring. There is a non-negligible overhead to running a test case that we need to try and reduce more.